src/cfn.py

```diff
@@ -300,9 +301,10 @@ def _download(path, destination):
 
 @task
 @requires_aws_stack
-def upload_file(stackname, local_path, remote_path, overwrite=False):
-    with stack_conn(stackname):
-        print('stack:', stackname)
+def upload_file(stackname, local_path, remote_path=None, overwrite=False, node=1):
+    remote_path = remote_path or os.path.join("/tmp", os.path.basename(local_path))
+    with stack_conn(stackname, node=node):
```

Comment:
didn't remember this supported `node`

It seems it is forced to `int` inside the function (`node` here will be a string)

Reply:
> didn't remember this supported node

`stack_conn` supporting a `node` parameter is something I introduced relatively recently - this year some time. I think it was so I could ssh into a specific node in one command

good catch with the typing. I'll fix that up